src/client/js/components/circle-chart.js

```diff
@@ -205,7 +228,7 @@ customElements.define('circle-chart', class extends HTMLElement {
         sliceOffset += percentage
 
         return acc
-      }, []).join('')}
+      }, init).join('')}
```

Comment:
This is an example of JSDoc being less ergonomic than using plain TS. The problem here is that the type of `[]` is inferred to be an empty array, whereas we later [push strings onto it](https://github.com/mozilla/blurts-server/blob/7a9d215474a0334e43c421e3b8b509052c80054d/src/client/js/components/circle-chart.js#L227). In TS, we could've just told it that it's an (empty) array of strings inline:

```suggestion
      }, [] as string[]).join('')}
```

However, in JSDoc, I have to (I think) pass it a variable, so that variable can have the type annotation: https://github.com/mozilla/blurts-server/blob/7a9d215474a0334e43c421e3b8b509052c80054d/src/client/js/components/circle-chart.js#L211-L212

The pattern is still the same, the only difference is the intermediate variable.